packages/element/package.json

```
@@ -29,8 +29,8 @@
 	"sideEffects": false,
 	"dependencies": {
 		"@babel/runtime": "^7.16.0",
-		"@types/react": "^18.0.21",
-		"@types/react-dom": "^18.0.6",
+		"@types/react": "18.2.79",
+		"@types/react-dom": "18.2.25",
```

Let's make sure to keep the version ranges here:

```suggestion
		"@types/react": "^18.2.79",
		"@types/react-dom": "^18.2.25",
```

Did the minimum version here need to be updated? If not, it's nicer for consumers to keep the wider range, we can still get this updated in Gutenberg because the newer package satisfies the range.

---

It was not mandatory, just wanted to keep things consistent. That's what we do for all dependencies so far  if I'm not wrong but your argument make sense.